sim/paladin/retribution/retribution.go

```diff
@@ -83,22 +89,23 @@ func (ret *RetributionPaladin) RegisterMastery() {
 
 	// Hand of Light
 	ret.HandOfLight = ret.RegisterSpell(core.SpellConfig{
-		ActionID:       actionId,
-		SpellSchool:    core.SpellSchoolHoly,
-		ProcMask:       core.ProcMaskMeleeMHSpecial,
-		Flags:          core.SpellFlagMeleeMetrics | core.SpellFlagIncludeTargetBonusDamage | core.SpellFlagNoOnCastComplete,
+		ActionID:    actionId,
+		SpellSchool: core.SpellSchoolHoly,
+		ProcMask:    core.ProcMaskMeleeMHSpecial,
+		Flags: core.SpellFlagMeleeMetrics |
+			// This flag makes sure 8% magic damage on target gets included as a multiplier
+			core.SpellFlagIncludeTargetBonusDamage |
```

Comment:
This comment is not true. That flag makes the spell include a very specific damage mod which is +bonus physical damage on target. You should remove this flag as its not corret on here.

From previous tests HoL should ignore both attacker and target modifiers and its only the 8% buff that affects it. Check Scourge Strike to see how to achieve that same effect

Reply:
My bad, will fix